dev/io.openliberty.microprofile.lra.1.0.internal/bnd.bnd

```diff
@@ -0,0 +1,53 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Long Running Actions
+Bundle-SymbolicName: io.openliberty.microprofile.lra.1.0.internal
+Bundle-Description: Long Running Actions Impl; version=${bVersion}
+
+Export-Package: io.openliberty.microprofile.lra.internal;provide:=true
+
+Private-Package: io.openliberty.microprofile.lra.internal.resources
+
+javac.source: 1.8
+javac.target: 1.8
+
+#Private-Package: com.ibm.ws.example.internal
+
+# This is how a component is declared in bnd (i.e. not through annotations)
+#Service-Component: \
+#  com.ibm.ws.lra;\
+#    implementation:=com.ibm.ws.lra.SampleComponent;\
+#    configuration-policy:=ignore;\
+#    properties:="service.vendor=IBM"
+    
+-dsannotations: \
+    io.openliberty.microprofile.lra.internal.LraFilterComponent, \
+    io.openliberty.microprofile.lra.internal.LraConfig
+
+Include-Resource: \
+  OSGI-INF=resources/OSGI-INF
+    
+-buildpath: \
+	io.openliberty.org.jboss.narayana.rts;version=latest,\
+	com.ibm.ws.logging.core;version=latest,\
+	com.ibm.websphere.org.osgi.core;version=latest,\
+	com.ibm.websphere.org.osgi.service.cm;version=latest,\
+	com.ibm.websphere.org.osgi.service.component;version=latest,\
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest, \
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest, \
+	com.ibm.websphere.javaee.jaxrs.2.0;version=latest, \
```

Comment:
Should this depend on jaxrs.2.1?

Reply:
I thought you need to depend on cdi as well.

Reply:
There is no dependency on cdi from this project.